### ISSUE TITLE
2019-12-01 tc014 M2SS no CEST_C0 values are generated for estimated concentration dataset

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#72]
With commit there are no values of CEST_C0 being generated in the estimated concentration dataset. However these values do appear in the parameter dataset where appropriate.

## output parameters are in the "r" dataframe

```
> summary(r$C0)
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA's 
  97400  170500  197000  229064  238500  604000      10 
> r[1:2,]
      SDEID DOSE1 DOSEC     C0       V01  CMAX1   CMAXC1   CMAXDN1 FLGACCEPTPREDOSE  CMIN1 CLAST1 TMAX1 TMIN1 CTROUGH1 CTROUGHEND1 PTROUGHR1         KEL KELTMLO KELTMHI KELNOPT    KELRSQ   KELRSQA
1 173902449  1160  1160 196000 0.1689655 585000 389262.8 504310345                0 196000 244000     1     0   196000      244000  2.984694 0.001341865     168     504       3 0.9130651 0.8261302
2 195715794    NA    NA     NA        NA     NA       NA        NA             <NA>     NA     NA    NA    NA       NA          NA        NA          NA      NA      NA      NA        NA        NA
  FLGACCEPTKEL    THALF THALFF LASTTIME FLGACCEPTTAU    AUCALL  AUCLAST1 AUCLASTC1 AUCLASTDN1   AUC1     AUC2     AUC3     AUC4      AUC5      AUC6 AUCINT1 AUCINT2 AUCINT3 AUCINT4 AUCINT5 AUCINT6
1            0 516.5549      1      504            1 179045785 179045785 -41293008   154.3498 390500 12635066 37300178 81430766 135825449 179045785     0_1    0_24    0_72   0_168   0_336   0_504
2         <NA>       NA   <NA>       NA         <NA>        NA        NA        NA         NA     NA       NA       NA       NA        NA        NA    <NA>    <NA>    <NA>    <NA>    <NA>    <NA>
    AUCINFO  AUCINFOC AUCINFODN   AUCINFP  AUCINFPC AUCINFPDN   AUCTAU1    AUMCTAU1 MRTIVIFO1 MRTIVIFP1 AUCXPCTO AUCXPCTP AUCXBPCTP     CAV1     CLTAU1 CLTAUW1     PTF1     PTR1    VSSP1 VSSPW1   VZTAU1
1 360882208 214816884  311.1054 353724187 207658863  304.9346 179045785 38049230822  38642785  37121605 50.38664 49.38266 0.1103967 355249.6 0.00647879      NA 1.095005 2.984694 240503.1     NA 4.828197
2        NA        NA        NA        NA        NA        NA        NA          NA        NA        NA       NA       NA        NA       NA         NA      NA       NA       NA       NA     NA       NA
  CONC1 CONC2 CONC3 CONC4 CONC5 CONC6 CONC7 CONCTIME1 CONCTIME2 CONCTIME3 CONCTIME4 CONCTIME5 CONCTIME6 CONCTIME7 DI1 TAU1 TOLD1 TIMEU AMOUNTU DOSEU VOLUMEU CONCU KELU  CLU     AUCU       AUMCU AUCNORMU
1   196   585   483   546   383   271   244         0         1        24        72       168       336       504  NA  504     0    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML NG.HR.HR/ML       NA
2    NA    NA    NA    NA    NA    NA    NA        NA        NA        NA        NA        NA        NA        NA  NA   NA    NA    HR      NA    MG       L NG/ML 1/HR L/HR NG.HR/ML NG.HR.HR/ML       NA
  AURCU CONCNORMU RATEU VOLUMEWU CLWU   SUBJID    STUDY SITEID RAND TRTCD PKCOLL PKBDFLD     PKTERM PERIODU PERIOD VISITU VISIT                     ACTTRT ACTTRTS PKPCOM
1    NA  NG/ML/MG    NA       NA   NA 10031003 A4021010   1003 3003  9999  POINT  Plasma CP-751,871   Cycle      4    Day     1 CP-751,871 dose escalation       A     NA
2    NA  NG/ML/MG    NA       NA   NA 10021001 A4021010   1002 2001  9999  POINT  Plasma CP-751,871   Cycle      4    Day     1 CP-751,871 dose escalation       A     NA
```

## Estimated concentration dataset values are stored in the "e" dataframe:

```
> head(e)
  PKDATAROWID    SDEID TIME CEST_KEL CEST_INT CEST_EXT CEST_C0 CEST_TLAST
1   -28860394 50896291   24 455.7413       NA       NA      NA         NA
2   -28860396 50896291   72 426.5652       NA       NA      NA         NA
3   -28860400 50896291  168 373.6968       NA       NA      NA         NA
4   -28860411 50896291  336 296.4517       NA       NA      NA         NA
5   -28860414 50896291  672 186.5619       NA       NA      NA   186.5619
6    28860267 51872317    1 908.3040       NA       NA      NA         NA
> e[e$SDEID==173902449,]
   PKDATAROWID     SDEID TIME CEST_KEL CEST_INT CEST_EXT CEST_C0 CEST_TLAST
18   -28860048 173902449  168 367.9231       NA       NA      NA         NA
19   -28860049 173902449  336 293.6653       NA       NA      NA         NA
20   -28860101 173902449  504 234.3949       NA       NA      NA   234.3949

> summary(e$CEST_C0)
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA's 
     NA      NA      NA     NaN      NA      NA      80 

```

**Comment by rudraya** [comment]
This issue was addressed in b8deb76, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
The estimated concentration dataset is now generated with commit b8deb76. However, the generated parameters do not appear to be correct. This will be reviewed and submitted separately. This issue can be closed.
